README.md

```diff
@@ -212,48 +195,47 @@ The following structure is to be finalized. The general convention for now is to
 ## Name Services
 
 - [NFDomains](https://nf.domains/) - Algorand name service and marketplace for Non-Fungible Domains (NFDs) — unique, readable aliases for wallet addresses.
-- [Algorand Name Service](https://algonameservice.com/home) - ANS is a decentralized naming service. An Algorand smart contract is used to store names and govern the naming policy.
+- [Algorand Name Service](https://algonameservice.com/home) - ANS is a decentralized naming service. An Algorand smart contract stores names and governs the naming policy.
 
 ## Community
 
-The following contains a set of sections related to open source projects, utilities and news resources.
+> The following contains sections related to open source projects, utilities, and news resources.
 
 ### Projects
 
 - [arc3.xyz](https://github.com/barnjamin/arc3.xyz) - Dapp that can be used to mint ARC3 compliant NFTs.
 - [Auction Demo](https://github.com/algorand/auction-demo) - On-chain NFT auction using smart contracts.
-- [AlgoMart](https://github.com/deptagency/algomart) - NFT storefront platform.
 - [Algorand Session Wallet](https://github.com/barnjamin/algorand-session-wallet) - Session wallet to allow persisted wallet connections across multiple wallets.
 - [AlgoWorld-Swapper](https://github.com/algoworldNFT/algoworld-swapper) - Collection of ASC1 contracts in pyteal for creating ASA swappers.
 - [WalletConnect Example DApp](https://github.com/algorand/walletconnect-example-dapp) - Algorand WalletConnect demo.
 - [paytx](https://github.com/algorand-devrel/paytx) - Examples of common pay transaction scenarios using the Algorand Python SDK.
 - [TinyBar App](https://tinybar.app) - A tiny macOS menu bar app for tracking ASA prices from TinyMan.
 - [Algodesk](https://www.algodesk.io/) - Create, configure and manage your Assets on Algorand Blockchain.
-- [algonim](https://github.com/cusma/algonim) - First Algorand mini-puzzle-game. Written in Python+PyTEAL by [@cusma](https://github.com/cusma).
+- [algonim](https://github.com/cusma/algonim) - First Algorand mini-puzzle-game. Written in Python+PyTEAL by [@cusma](https://twitter.com/cusma_b).
```

Comment:
Sorry @cusma i had to update it to avoid double-link warning in awesome-lint.

Reply:
I think you should modify the `algorealm` double link too, isn't it?

Reply:
@cusma actually i think double link rule in awesome-lint was checking for link duplicates, so thats why removing duplicated url to your gh profile fixed it. Otherwise all good now - you can see that all awesome-lint rules pass except for few manually silenced rules in the readme header (that will be removed in coming days as we become fully compliant)